temperature_regulator/project/test/test_txt_config_loader.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+import tempfile
+from main.config_loader.txt_config_loader import TxtConfigLoader
+
+
+class TxtConfigLoaderTestCase(unittest.TestCase):
+    def setUp(self):
+        self._file = tempfile.NamedTemporaryFile(mode="w+t")
+
+    def tearDown(self):
+        self._file.close()
+
+    def add_data_to_file(self, data: str):
+        self._file.write(data)
+        self._file.seek(0)
+
+    def test_listenerSocketAddress_EmptyFile_ThrowsKeyError(self):
+        data = ""
+        self.add_data_to_file(data)
+        config = TxtConfigLoader(self._file)
+        self.assertRaises(KeyError, lambda: config.listener_socket_address)
```

Comment:
jeżeli się nie mylę to można zapisać ` self.assertRaises(KeyError, config.listener_socket_address)`, ale nie jestem do końca pewien